core/commands/quick_stage.py

```diff
@@ -12,7 +12,8 @@
 CLEAN_WORKING_DIR = "Nothing to commit, working directory clean."
 ADD_ALL_UNSTAGED_FILES = " ?  All unstaged files"
 ADD_ALL_FILES = " +  All files"
-
+STAGED = "--- {} files are staged for commit. ---"
+COMMIT = "    git: commit"
 
```

Comment:
Nit: Add an extra line here.  PEP8 calls for two lines in-between top-level class definitions.